charts/scalardb-analytics-server/templates/scalardb-analytics-server/deployment.yaml

```diff
@@ -22,7 +22,9 @@ spec:
         {{- toYaml .Values.scalarDbAnalyticsServer.podAnnotations | nindent 8 }}
         {{- end }}
       labels:
-        {{- include "scalardb-analytics-server.selectorLabels" . | nindent 8 }}
+        {{- $podLabels := .Values.scalarDbAnalyticsServer.podLabels | default dict }}
+        {{- $selectorLabels := include "scalardb-analytics-server.selectorLabels" . | fromYaml }}
+        {{- toYaml (merge $selectorLabels $podLabels) | nindent 8 }}
```

Comment:
You can see `How it works` in the following comment in another PR.
- https://github.com/scalar-labs/helm-charts/pull/304#discussion_r2289955576